src/Common/src/Interop/Windows/mincore/Interop.LUIDAndAttributes1.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class mincore
+    {
+        internal unsafe partial struct InlineArray_LUID_AND_ATTRIBUTES_1
+        {
+            public LUID_AND_ATTRIBUTES this[uint index]
+            {
+                get
+                {
+                    if (index < 0
+                                || index >= 1)
+                        throw new global::System.IndexOutOfRangeException();
+                    fixed (InlineArray_LUID_AND_ATTRIBUTES_1* pThis = &(this))
+                        return ((LUID_AND_ATTRIBUTES*)pThis)[index];
```

Comment:
I guess this is supposed to be the same code for other similar structs, like InlineArray_LUID_AND_ATTRIBUTES_2/3/4, which would have more than one element? In this case though, the code doesn't really make much sense because there's only one element.